src/systemctl/systemctl-enable.c

```diff
@@ -267,7 +267,7 @@ int verb_enable(int argc, char *argv[], void *userdata) {
                 /* If some of the units are disabled in user scope but still enabled in global scope,
                  * we emit a warning for that. */
 
-                _cleanup_strv_free_ char **enabled_in_global_scope = NULL;
+                _cleanup_free_ char **enabled_in_global_scope = NULL;
```

Comment:
this is a bit icky, and given this is not performance critical here i'd avoid it. But if you must then please add a big fat comment about this, because this is irregular memory management, and people might see this later and assume it is a bug, even though it is not.